node_modules/gh-core/lib/api.js

```diff
@@ -192,3 +205,14 @@ var _initialiseExpressServers = function(config, callback) {
         DocsAPI.documentModule(module, moduleDocumented);
     });
 };
+
+/**
+ * Check if a URL is whitelisted from the Terms and Conditions requirements
+ *
+ * @param  {String}     url     The URL to check
+ * @return {Boolean}            `true` if the user doesn't have to accept the Terms and Conditions in order to POST to this url, `false` otherwise
+ * @api private
+ */
+var _isWhiteListed = function(url) {
```

Comment:
Can you give a more meaningful name to this function? "whitelist" is a bit too generic. Maybe `_requiresTermsAndConditions`